README.md

```diff
@@ -1,6 +1,86 @@
-# silk-paper
-Package for the automatic generation of REST documentation based on the OpenApi specification. Key name: Silk
+# silk-paper-node
+Package for the automatic generation of REST documentation based on the OpenApi specification.
+
+## Installing
+Install with `npm i @wolox/silk-paper`.
+
+## Usage
+* In `app.js`:
+We are using [`swagger-ui-express`](https://www.npmjs.com/package/swagger-ui-express) package for render the documentation, so first you have to install it.
```

Comment:
Don't say `we are using`, remember that this part doesn't depend on us. Leave it as an example of how to use it. But not as if the package depends on swagger-ui.